dev/app/init-builtin-widgets.js

```diff
@@ -269,6 +269,25 @@ export default () => {
           })
         );
       })
+    )
+    .add(
+      'with clearsQuery',
+      wrapWithHits(
+        container => {
+          window.search.addWidget(
+            instantsearch.widgets.currentRefinedValues({
+              container,
+              clearsQuery: true,
+            })
+          );
+        },
+        {
+          searchParameters: {
+            disjunctiveFacetsRefinements: { brand: ['Apple', 'Samsung'] },
+            disjunctiveFacets: ['brand'],
+          },
+        }
+      )
     );
```

Comment:
Since you're modifying the template, can you add a story with the different kind of parameters? So that we can check that everything is going well?